.buildkite/pipeline.build.yml

```diff
@@ -202,12 +202,18 @@
 - label: ":python: (Large)"
   conditions: ["RAY_CI_PYTHON_AFFECTED"]
   instance_size: large
-  parallelism: 3
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
     - DL=1 ./ci/env/install-dependencies.sh
     - ./ci/env/env_info.sh
-    - ./ci/ci.sh test_large
+    - bazel test --config=ci $(./ci/run/bazel_export_options) --test_env=CONDA_EXE --test_env=CONDA_PYTHON_EXE
+      --test_env=CONDA_SHLVL --test_env=CONDA_PREFIX --test_env=CONDA_DEFAULT_ENV --test_env=CONDA_PROMPT_MODIFIER
+      --test_env=CI --test_tag_filters="large_size_python_tests_shard_0,large_size_python_tests_shard_1,large_size_python_tests_shard_2"
+      -- $(yq -r .flaky_tests[] ci/v2/test/core.tests.yml)
```

Comment:
is it true that at the end of this series of changes, the `yq` will go away where we will be parsing these yml files with `ray_ci` python script?

Reply:
that's correct yes, getting there